python/lsst/cp/pipe/astierCovPtcFit.py

```diff
@@ -236,12 +235,13 @@ class CovFit:
         Maximum range to select from tuple.
     """
 
-    def __init__(self, inputTuple, maxRangeFromTuple=8):
+    def __init__(self, inputTuple, maxRangeFromTuple=8, expIdMask=None):
```

Comment:
`expIdMask` needs to be listed in the parameters above.  I'm also not sure if `None` is the correct default.  I think the usage here will cause an error ('list indices must be integers or slices, not NoneType').
Also, if this is unrelated to the `expIdMask` used elsewhere (which is a dict containing amp names), can this be called something else?